packages/st2mistral/debian/mistral-api.init

```diff
@@ -20,7 +20,8 @@ DESC="mistral"
 NAME=mistral-api
 DAEMON=/opt/stackstorm/mistral/bin/gunicorn
 COMPONENTS="api,engine,executor"
-PIDFILE=/var/run/mistral/$NAME.pid
+RUN_DIR="/run/mistral"
```

Comment:
Use `/var/run` (old compatible way), because a symlink will be in place to `/run` if applicable.